kubernetes/kubernetes_meter.py

```diff
@@ -44,13 +44,17 @@ def _parse_apel_str(self, apel_record: str):
         self.apel_dict = {}
         lines = apel_record.split('\n')
         for line in lines:
-            kv_pair = [v.strip() for v in line.split(':')]
-            if len(kv_pair) == 2:
-                self.apel_dict[kv_pair[0]] = kv_pair[1]
+            # Handle case where apel values contain colons (eg. 'SubmitHost: http://localhost:8000')
+            kv_pair_match = re.match(r'([A-Za-z0-9_]+)\s*:\s*([^\s].*[^\s])\s*',line)
```

Comment:
How about just limiting the number of splits?

```
>>> 'SubmitHost: http://localhost:8000'.split(':', maxsplit=1)
['SubmitHost', ' http://localhost:8000']
```

Reply:
Didn't know about the maxsplit kwarg, that's very useful